### ISSUE TITLE
Input issue in empty block on mobile

### ISSUE THREAD

**Issue by johanneswilm** [ProseMirror/prosemirror#362]
Error shows when using no Pm-internal menu or menuBar menu. Tried on Mobile Safari 9.2&9.3 on Ipad as well as Chrome on Android 6.0 (PM 0.7.0):

For example, entering a letter in a new document (with no text, just standard elements) should give a step of:

```
stepType: replace
from: 14
to: 14
slice: {
    ...
}
structure: false
```

But instead creates:

```
stepType: replace
from: 12
to: 12
slice: null
structure: false
```

When then trying to resolve this from-value a RangeError is produced.

 The error seems to go away when adding the tooltip menu:

```
import "prosemirror/dist/menu/tooltipmenu"
....
pm.setOption("tooltipMenu", {selectedBlockMenu: true})
```

**Comment by johanneswilm** [comment]
Suddenly I cannot reproduce how to make it work in empty blocks any more. It generally doesn't seem to allow input on mobile in empty text blocks, but there are fewer error messages about it when the tooltip menu is active.

**Comment by marijnh** [comment]
I set up the demo to start with an empty paragraph as content and disabled the menus, but on Android Chrome, typing into that empty paragraph (both when it is the only content in the document and when it exists between other content) seems to work fine.

**Comment by johanneswilm** [comment]
Yeah, sorry. I'm currently trying to isolate the issue. I thought I had it yesterday, but further testing proved me wrong.

**Comment by johanneswilm** [comment]
@marijnh OK, I have it isolated. I modified your demo as of PM 0.7.0 in a way to show the bug[1]. I gave it a scheme like this:

    let  demoSchema = new Schema({
      nodes: {
        doc: {type: Doc, content: "title body"},
        title: {type: Title, content: "text*", group: "part"},
        body: {type: Body, content: "block+", group: "part"},
        ... // like default schema
      }
      ...
    }

And an initial document like this:

    <doc><title></title><body></body></doc>

Test these: 

* On mobile, type anything into the body
* On mobile, try to delete something from the title 

I assume this has something to do with initial setup on page load, because if I load the demo editor in Chrome desktop and then switch to ipad view, it works. But if I first switch to ipad view, then load the demo, it doesn't work. Even if I switch back to desktop view, it will not take any input.

[1] https://github.com/fiduswriter/prosemirror/tree/mobile-bug
